main.py

```diff
@@ -29,8 +29,11 @@
 num_updates = int(args.num_frames) // args.num_steps // args.num_processes
 
 torch.manual_seed(args.seed)
-if args.cuda:
-    torch.cuda.manual_seed(args.seed)
+torch.cuda.manual_seed_all(args.seed)
```

Comment:
Is this line still correct when cuda is not available? Or line 32 is still necessary?

Reply:
According to the pytorch documentation (https://pytorch.org/docs/stable/cuda.html):

torch.cuda.manual_seed_all(seed)
"Sets the seed for generating random numbers on all GPUs. It’s safe to call this function if CUDA is not available; in that case, it is silently ignored."

So i would think line 32 is not necessary anymore.

Reply:
Great! Thanks. Then I'm merging.